x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -367,7 +367,9 @@ function FieldItemPopoverContents(props: FieldItemProps) {
           }
 
           if (params?.noDataFound) {
-            const isUsingSampling = core.uiSettings.get('lens:useFieldExistenceSampling');
+            const isUsingSampling = core.uiSettings.get(
+              'unifiedFieldList:useFieldExistenceSampling'
```

Comment:
This change is not backwards compatible. As we are going to remove this setting in 8.6 anyway, let's keep it under the old name.